go/builder.go

```diff
@@ -575,11 +606,12 @@ func (b *Builder) FinishWithFileIdentifier(rootTable UOffsetT, fid []byte) {
 }
 
 // Finish finalizes a buffer, pointing to the given `rootTable`.
-func (b *Builder) Finish(rootTable UOffsetT) {
+func (b *Builder) Finish(rootTable UOffsetT) *Builder {
```

Comment:
@tsingson I still don't see why this change needs to be part of this PR. Could you explain further, please? :-]

Reply:
I thought about it some time, and yes, the chain call is not necessary for most people.
I'll remove this change later.